src/anomalib/visualization/image/functional.py

```diff
@@ -0,0 +1,408 @@
+
+    Raises:
+        TypeError: If the mask is not a PIL Image or PyTorch tensor.
+
+    Examples:
+        1. Visualize a PIL Image mask:
+        >>> from PIL import Image
+        >>> import numpy as np
+        >>> mask_array = np.random.randint(0, 256, size=(100, 100), dtype=np.uint8)
+        >>> mask_image = Image.fromarray(mask_array)
+        >>> colored_mask = visualize_mask(mask_image, color=(255, 0, 0))  # Red mask
+        >>> colored_mask.show()
+
+        2. Visualize a PyTorch tensor mask:
+        >>> import torch
+        >>> mask_tensor = torch.randint(0, 2, size=(100, 100), dtype=torch.bool)
+        >>> colored_mask = visualize_mask(mask_tensor, color=(0, 255, 0))  # Green mask
+        >>> colored_mask.show()
+
+        3. Visualize a grayscale PyTorch tensor mask:
+        >>> mask_tensor = torch.randint(0, 256, size=(100, 100), dtype=torch.uint8)
+        >>> colored_mask = visualize_mask(mask_tensor, color=(0, 0, 255))  # Blue mask
+        >>> colored_mask.show()
+
+        4. Visualize without applying color:
+        >>> mask_array = np.random.randint(0, 256, size=(100, 100), dtype=np.uint8)
+        >>> mask_image = Image.fromarray(mask_array)
+        >>> grayscale_mask = visualize_mask(mask_image)
+        >>> grayscale_mask.show()
+    """
+    if isinstance(mask, torch.Tensor):
+        if mask.dtype == torch.bool:
+            mask = mask.to(torch.uint8) * 255
+        mask = to_pil_image(mask)
+
+    if not isinstance(mask, Image.Image):
+        msg = "Mask must be a PIL Image"
+        raise TypeError(msg)
+
+    if color:
+        # Create a solid color image
+        solid_color = Image.new("RGB", mask.size, color)
+
+        # Use the original mask as alpha
+        mask = Image.composite(solid_color, Image.new("RGB", mask.size, (0, 0, 0)), mask)
+
+    return mask
+
+
+def create_image_grid(images: list[Image.Image], nrow: int) -> Image.Image:
+    """Create a grid of images using PIL.
+
+    Args:
+        images: List of PIL Images to arrange in a grid.
+        nrow: Number of images per row.
+
+    Returns:
+        A new PIL Image containing the grid of images.
+    """
+    if not images:
+        msg = "No images provided to create grid"
+        raise ValueError(msg)
+
+    # Assuming all images have the same size
+    img_width, img_height = images[0].size
+
+    # Calculate grid dimensions
+    ncol = (len(images) + nrow - 1) // nrow  # Ceiling division
+    grid_width = nrow * img_width
+    grid_height = ncol * img_height
+
+    # Create a new image with white background
+    grid_image = Image.new("RGB", (grid_width, grid_height), color="white")
+
+    # Paste images into grid
+    for idx, img in enumerate(images):
+        row = idx // nrow
+        col = idx % nrow
+        grid_image.paste(img, (col * img_width, row * img_height))
+
+    return grid_image
+
+
+def visualize_field(
+    field: str,
+    value: torch.Tensor,
+    *,  # Mark the following arguments as keyword-only
```

Comment:
Where is this used?

Reply:
I want user to use keywords for the rest of the arguments after *. For example, following line follows this keyword-only format 
https://github.com/samet-akcay/anomalib/blob/c17c77ce655cc146e065699bd10f69a9cb9b3ac1/src/anomalib/visualization/image/functional.py#L376

Reply:
```python
image = visualize_field(field, value, colormap=colormap, normalize=normalize)
```

Reply:
So, basically we want to force the users to pass parameters using keywords

Reply:
Yes, because such args are hard to read/follow if a keyword is not provided